### ISSUE TITLE
isResubscribe field is no longer present on subscribed event

### ISSUE THREAD

**Issue by nmorel** [centrifugal/centrifuge-js#213]
**Is your feature request related to a problem? Please describe.**
In v2 client, there was a field `isResubscribe` given on subscribe event.  
In v3, it's no longer present. If history is enabled, I think I could use `wasRecovering` or `recovered`.

**Describe the solution you'd like**
It would be nice to have the field back.  

**Describe alternatives you've considered**

- Add listener only after the first successful subscribe
- Add some kind of counter that increments on each call. If superior to 1, it's a resubscribe.

**Comment by FZambia** [comment]
Hello, the field is removed, correct.

`isResubscribe` was confusing – it's hard to understand which business task it was aimed to solve. Current fields match directly to client protocol fields. So I don't want adding it back if it's possible to solve the problem on top of SDK functionality. What is the issue you can't resolve with current fields? It seems you suggested a couple of alternative approaches which can solve the issue.

**Comment by nmorel** [comment]
Understood 👌  
We don't have history enabled yet and we use this field to refetch or refresh our data in case we missed something.  
I'm gonna use one of the alternative 🙂
